packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -165,7 +165,9 @@ const BorderControlDropdown = (
 		? 'bottom left'
 		: undefined;
 
-	const renderToggle = ( { onToggle = noop } ) => (
+	const renderToggle: DropdownComponentProps[ 'renderToggle' ] = ( {
+		onToggle,
```

Comment:
`renderToggle` always has an `onToggle` prop, so no need for the `noop` default.